addons/website_livechat/static/tests/helpers/mock_models.js

```diff
@@ -17,9 +17,19 @@ MockModels.patch('website_livechat/static/tests/helpers/mock_models.js', T =>
             const data = super.generateData(...arguments);
             Object.assign(data, {
                 'website.visitor': {
-                    fields: {},
+                    fields: {
+                        country_id: { string: "Country", type: 'many2one', relation: 'res.country' },
+                        display_name: { string: "Display name", type: 'string' },
+                        // Should not be a field as it's a string normally computed by server based on user
+                        // navigation. To ease testing this allows tests to set it directly.
+                        history: { string: "History", type: 'string'},
```

Comment:
Would use the _"what"_ then _"why"_ then _"extra info"_ format for the docstring of this mock server data field definition.

It feels very aggressive to start with the extra info, which is reinforced by hard-to-read 1st sentence (too dense in my opinion), and 2nd sentence slightly hidden (by not being at the start of comment line).